src/test/unit/pages/LibrariesPage.test.tsx

```diff
@@ -10,7 +10,7 @@ let setup = () => {
   getLibrariesStub.mockResolvedValue([{id: "123",
     name: "Want to Play",
     games: [{
-        id: "123",
+        id: "1",
```

Comment:
Fixing React unique key issue - filed a ticket to address this elsewhere in the codebase https://gaming-backlog.atlassian.net/browse/GB-83